aten/src/ATen/native/Copy.cpp

```diff
@@ -163,6 +164,13 @@ Tensor& copy_(Tensor& self, const Tensor& src, bool non_blocking) {
   return self;
 }
 
+static auto registry = torch::RegisterOperators()
+  .op(torch::RegisterOperators::options()
+    .schema("aten::copy_(Tensor(a!) self, Tensor src, bool non_blocking=False) -> Tensor(a!)")
+    .impl_unboxedOnlyCatchAllKernel<decltype(copy_), &copy_>()
+    .aliasAnalysis(AliasAnalysisKind::FROM_SCHEMA))
+  ;
```

Comment:
Why does `copy_` need manual registration? It seems to me that the normal mechanism will register a catchAll, as required, and then you can manually register the Variable implementation without having to do anything extra.

Reply:
Ditto with `resize` and the other methods for whom we manually register VariableTypeId implementations

Reply:
The normal mechanism will register a catchAll **and** a variable implementation. We need to skip registering the variable implementation from codegen so we can plug in our own. This is why `copy_` and `resize_` have been part of the `MANUAL_IMPLEMENTATIONS` setup in `gen_variable_type.py`, which is removed by this PR and replaced with `manual_kernel_registration`.